plugins/field-date/test/test.mocha.js

```diff
@@ -0,0 +1,159 @@
+/**
```

Comment:
Other tests in samples like workspace search and slider are using the naming convention for test to be:
field_date_test.mocha.js
I'm not sure if just test or the longer name is better, could update the other ones. I'd just like to be consistent

Reply:
Sure, happy to switch.